src/App.vue

```diff
@@ -11,17 +11,16 @@ declare var AlgoSigner: any; // eslint-disable-line
 
 export default defineComponent({
 	name: "App",
-	components: {},
 	data() {
 		return {
 			walletCheckCount: 1,
 		};
 	},
 	setup() {
-		const counter = WalletStore();
+		const walletStore = WalletStore();
 
 		return {
-			setHasAlgoSigner: counter.setHasAlgoSigner,
+			setHasAlgoSigner: walletStore.setHasAlgoSigner,
```

Comment:
Why we are using "set algo signer"? What if a user will like to use WalletConnect.

I think we should have a function `walletStore.connect(type, ...)`, where type is something like (I'm sure we already have some enum for that):
```
enum WalletType {
   WalletConnect
   AlgoSigner
   MyAlgo
}
```

PS: if this requires an update in the web package, then please add appropriate tasks (one for the web package and one task for the sigma dao). This is a high priority - because we want to release the web package soon. so we need to make sure we have all required API to connect to all wallets.

Reply:
This enum already exists. Yes, you are right I should give the right function name and make it setWalletType

Reply:
Exactly, that's why I wrote that _I'm sure we already have ...._